### ISSUE TITLE
Are there any plans to expanding to italics?

### ISSUE THREAD

**Issue by dburov190** [coz-m/MPLUS_FONTS#16]
Hi! I really like M+ fonts, but on many occasions I'd like to use italics for emphasis. Are you considering expanding the fonts to include italic types?

Thanks!

**Comment by coz-m** [comment]
I also have a desire to align italics and complete it as a Latin font, but before that I would like to try some Japanese Kana variations and add the missing Kanji glyphs, so I think it will be a while later.

**Comment by ttyrakow** [comment]
I humbly join the request by @dburov190. Combining M PLUS 2 with M PLUS Code Latin 60 in an IDE (like IntelliJ or Godot) creates a unique experience. Italics would be a great addition. Unfortunately, my experience with font creation is next to none (although I can imagine how much work it requires), so I can only wait patiently, hoping you'll eventually add italics to your coding fonts (which are great nonetheless, italics or not).

**Comment by JordanAnthonyKing** [comment]
So long as it's not a cursive latin font like so many hipster coding fonts are wont to do recently :L

**Comment by ttyrakow** [comment]
No, nothing like operator mono or such! I had a dedicated italic version of the font in mind (the automatic "italization" of a straight font made by some editors doesn't look well).

**Comment by pkazmier** [comment]
Likewise. M PLUS Code is my new favorite programming font. It’s easily the most legible and stylish font I’ve used. I, too, look forward to real italics (not script-like) someday. Thanks for your efforts on such a beautiful font.

**Comment by cmarincia** [comment]
Thank you for this amazing font set. I would really like an italic version of the font.

**Comment by Zaeroses** [comment]
I'll look into generating italic font files for personal use for the time being, but I would also appreciate a (readable) italic version!  And, like others say, script versions for italics don't work (certainly not for monospace fonts), and they make it really quite hard for me to read... the reason I use M+ fonts is specifically for their readability over nearly any other monospace font I have found!

For many things, automatic 'italization' does work - I feel like it's only certain letters that require italic versions, like `a, f, i, j, k, q, x, y` perhaps...?  I don't know, I am no font designer!
